common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -44,10 +44,29 @@ export const stringToBuffer = (input: string, encoding: string): ArrayBufferLike
 export const bufferToString = (blob: ArrayBufferLike, encoding: string): string =>
      IsoBuffer.from(blob).toString(encoding);
 
+/**
+ * Determines if an object is an array buffer
+ * @param obj - The object to determine if it is an ArrayBuffer
+ */
+export function isArrayBuffer(obj: any): obj is ArrayBuffer {
+    const maybe = obj as Partial<ArrayBuffer> | undefined;
+    return obj instanceof ArrayBuffer
+    || (typeof maybe === "object"
+        && typeof maybe.byteLength === "number"
+        && typeof maybe.slice === "function");
+}
+
 /**
  * Minimal implementation of Buffer for our usages in the browser environment.
  */
 export class IsoBuffer extends Uint8Array {
+    private readonly ___isIsoBuffer___  = true;
+
+    public static isIsoBuffer(obj: any): obj is IsoBuffer {
+        const maybe = obj as IsoBuffer;
+        return maybe.___isIsoBuffer___ === true;
+    }
+
     // Need to have ctor for it to be in proto chain for instanceof check in from() method to work
```

Comment:
You can remove ctor below.

Reply:
question. do we pass these across the loader container boundary? as this change won't be back compat